homework-2/essay.md

```diff
@@ -0,0 +1,14 @@
+# The best holidays in my life!
+
+Hey... I'm still Dmitry, and I don't have a classification of my holidays. So today I tell you about my holidays in Spain.
+
+It was randomly. I and my friend wanted to spend the holidays outside of Russia and we looked for a tour with all-inclusive. We found a tour to Spain, to Mallorca. We bought a tour, made a visa and started to prepare.
+
+When we flew there and came to the hotel we found out that we live on the German coast. As it turned out tourist Mallorca divided on the two coasts - a German coast and an English coast. We bought two excursions - "A journey for Mallorca" and "Sea caves".
```

Comment:
- lived on
- is divided (it is passive, it cannot divide itself)
- do not use articles before numbers (on two coasts)